articles/keeping-track-of-yubikeys.md

```diff
@@ -0,0 +1,40 @@
+# Keeping track of YubiKeys
```

Comment:
Maybe a more juicy title?

YubiTrak: Easy YubiKey retrieval with Apple Airtags
YubiTrak: Lossless YubiKeys with Airtags

Reply:
Will go with `Lossless YubiKeys with Yubitrak and AirTag` - thanks!